lib/components/useToast/useToast.docs.tsx

```diff
@@ -15,13 +15,11 @@ import { useThemeSettings } from '../../../site/src/App/ThemeSetting';
 import Code from '../../../site/src/App/Code/Code';
 import Toast, { toastDuration } from './Toast';
 import source from '../../utils/source.macro';
-import { useToast } from './ToastContext';
 
 const docs: ComponentDocs = {
   category: 'Content',
-  Example: ({ id }) => {
+  Example: ({ id, showToast }) => {
     const { theme } = useThemeSettings();
-    const showToast = useToast();
```

Comment:
It seems like the example doesn't use `useToast` anymore.

Reply:
A little confusing i admit. If you expand the code section on the docs it doesnt show the code that was changed here, rather it shows the usage, e.g. `showToast`. This is consistent with the other examples down the page too. The reason for this is so the code can be copied into Playroom (where there are limitations around Hook usage), but I agree it doesnt do the documentation any favours.

Further down under [Development Considerations](https://seek-oss.github.io/braid-design-system/components/useToast#development-considerations) we show how to use the Hook, and talk about `showToast` under [Playroom Prototyping](https://seek-oss.github.io/braid-design-system/components/useToast#playroom-prototyping).

I have a few ideas though that we can chat about offline.